### PR TITLE
Adding functions ban, unban in users.py

### PR DESCRIPTION
**Pull request by ProgramUser5353**

### CODE REVIEW

**Comment by Olegt0rr** [comment]
Please STOP.
I'm beggin

File: aiogram/types/user.py [hunk 1]
```diff
@@ -86,6 +86,18 @@ async def get_user_profile_photos(self, offset=None, limit=None):
 
     async def get_profile_photos(self, offset=None, limit=None):
         return await self.bot.get_user_profile_photos(self.id, offset, limit)
+    
+    async def ban(self, until_date: typing.Union[base.Integer, datetime.datetime,
+                                                         datetime.timedelta, None] = None,
+                  revoke_messages: typing.Optional[base.Boolean] = None
+                  ) -> base.Boolean:
+        return await self.bot.ban_chat_member(self.chat.id, self.id, until_date=until_date,
```
Comment (evgfilim1) [inline aiogram/types/user.py:94]: `AttributeError: 'User' object has no attribute 'chat'`

File: aiogram/types/user.py [hunk 2]
```diff
@@ -86,6 +86,18 @@ async def get_user_profile_photos(self, offset=None, limit=None):
...
+                                              revoke_messages=revoke_messages)
+
+    async def unban(self, only_if_banned: Optional[base.Boolean]) -> base.Boolean:
+        return await self.bot.unban_chat_member(self.chat.id,
```
Comment (evgfilim1) [inline aiogram/types/user.py:98]: `AttributeError: 'User' object has no attribute 'chat'`

**Review by evgfilim1 — CHANGES_REQUESTED** [review]
Also, please fill in pull request template, don't ignore it!

**Comment by evgfilim1** [comment]
You cannot implement these methods on User object, as ban and unban requires chat_id. It even doesn't really fit here semantically. Ban and unban are more appropriate for Chat object, but they already exist there. Closing as won't fix.

**Comment by ProgramUser5353** [comment]
> You cannot simply implement these methods on User object, as ban and unban requires chat_id. It even doesn't really fit here semantically. Ban and unban are more appropriate for Chat object, but they already exist there. Closing as won't fix.

but it is possible to get value of ID from users somehow?

**Comment by evgfilim1** [comment]
> but it is possible to get value of ID from users somehow?

User ID is known, but in order to ban the user, you have to provide a chat_id in which you want to ban him.

**Pull request closed without merging**